dataplane/vrf.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <array>
+
+#include "lpm.h"
+#include "type.h"
+
+namespace dataplane::vrflpm
+{
+
+template<typename Address, typename InnerLpmType>
+class VrfLookuper
+{
+public:
+	void Update(std::array<InnerLpmType*, YANET_RIB_VRF_MAX_NUMBER> lpms)
+	{
+		for (size_t index = 0; index < lpms_.size(); index++)
+		{
+			lpms_[index] = lpms[index];
+		}
+	}
+
+	void Lookup(const Address* ipAddresses, const tVrfId* vrfIds, uint32_t* valueIds, const unsigned int& count) const
+	{
+		for (unsigned int index = 0; index < count; index++)
+		{
+			valueIds[index] = lpmValueIdInvalid;
+			tVrfId vrf = vrfIds[index];
+			if ((vrf < YANET_RIB_VRF_MAX_NUMBER) && (lpms_[vrf] != nullptr))
+			{
+				lpms_[vrf]->lookup(ipAddresses + index, valueIds + index, 1);
+			}
+		}
+	}
```

Comment:
The correct approach would be to use iterators to `std::array` instead of pointer arithmetics, but we're so elbow deep in c-style arrays and pointer arithmetics in `worker.h` and in underlying `lpm` that such change would seem unnecessary.
Let's hope that in YANET2 we will be able to make things right (:

(that's not an issue, ignore it)